datasets/multi_re_qa/multi_re_qa.py

```diff
@@ -0,0 +1,227 @@
+    author={Mandy Guo and Yinfei Yang and Daniel Cer and Qinlan Shen and Noah Constant},
+    year={2020},
+    eprint={2005.02507},
+    archivePrefix={arXiv},
+    primaryClass={cs.CL}
+}"""
+# You can copy an official description
+_DESCRIPTION = """MultiReQA contains the sentence boundary annotation from eight publicly available QA datasets including SearchQA, TriviaQA, HotpotQA, NaturalQuestions, SQuAD, BioASQ, RelationExtraction, and TextbookQA. Five of these datasets, including SearchQA, TriviaQA, HotpotQA, NaturalQuestions, SQuAD, contain both training and test data, and three, including BioASQ, RelationExtraction, TextbookQA, contain only the test data"""
+
+_HOMEPAGE = "https://github.com/google-research-datasets/MultiReQA"
+
+# License for the dataset is not available
+_LICENSE = ""
+
+# Official links to the data hosted on github are below
+# Train and Dev sets are available only for SearchQA, TriviaQA, HotpotQA, SQuAD and NaturalQuestions
+# Test sets are only available for BioASQ, RelationExtraction and TextbookQA
+
+train_SearchQA = (
+    "https://github.com/google-research-datasets/MultiReQA/raw/master/data/train/SearchQA/candidates.json.gz"
+)
+dev_SearchQA = "https://github.com/google-research-datasets/MultiReQA/raw/master/data/dev/SearchQA/candidates.json.gz"
+
+train_TriviaQA = (
+    "https://github.com/google-research-datasets/MultiReQA/raw/master/data/train/TriviaQA/candidates.json.gz"
+)
+dev_TriviaQA = "https://github.com/google-research-datasets/MultiReQA/raw/master/data/dev/TriviaQA/candidates.json.gz"
+
+train_HotpotQA = (
+    "https://github.com/google-research-datasets/MultiReQA/raw/master/data/train/HotpotQA/candidates.json.gz"
+)
+dev_HotpotQA = "https://github.com/google-research-datasets/MultiReQA/raw/master/data/dev/HotpotQA/candidates.json.gz"
+
+train_SQuAD = "https://github.com/google-research-datasets/MultiReQA/raw/master/data/train/SQuAD/candidates.json.gz"
+dev_SQuAD = "https://github.com/google-research-datasets/MultiReQA/raw/master/data/dev/SQuAD/candidates.json.gz"
+
+train_NaturalQuestions = (
+    "https://github.com/google-research-datasets/MultiReQA/raw/master/data/train/NaturalQuestions/candidates.json.gz"
+)
+dev_NaturalQuestions = (
+    "https://github.com/google-research-datasets/MultiReQA/raw/master/data/dev/NaturalQuestions/candidates.json.gz"
+)
+
+test_BioASQ = "https://github.com/google-research-datasets/MultiReQA/raw/master/data/test/BioASQ/candidates.json.gz"
+
+test_RelationExtraction = (
+    "https://github.com/google-research-datasets/MultiReQA/raw/master/data/test/RelationExtraction/candidates.json.gz"
+)
+
+test_TextbookQA = (
+    "https://github.com/google-research-datasets/MultiReQA/raw/master/data/test/TextbookQA/candidates.json.gz"
+)
+
+test_DuoRC = "https://github.com/google-research-datasets/MultiReQA/raw/master/data/test/DuoRC/candidates.json.gz"
+
+
+class Multi_Re_Qa(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class MultiReQa(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `multi_re_qa`